src/transports/ipc/aipc.c

```diff
@@ -89,6 +90,14 @@ void nn_aipc_start (struct nn_aipc *self, struct nn_usock *listener)
     self->listener_owner.fsm = &self->fsm;
     nn_usock_swap_owner (listener, &self->listener_owner);
 
+#if defined NN_HAVE_WINDOWS
+    /* Get/Set security attribute pointer*/
+    nn_epbase_getopt (self->epbase, NN_SOL_SOCKET, NN_SEC_ATTR, &self->usock.sec_attr, &sz);
```

Comment:
Is it ok if they will be still defined in `sock.h` and the processing will be done in `sock.c`?
I'm not familiar with the project structure that much.